README.md

```diff
@@ -103,6 +109,7 @@ Download the repos file that will be used to download Fast RTPS and its dependen
 $ wget https://raw.githubusercontent.com/eProsima/Fast-DDS/master/fastrtps.repos
 $ mkdir src
 $ vcs import src < fastrtps.repos
+$ cd src
```

Comment:
You do not need to change directory to `src`. Colcon will scan the directory and find the packages to build. That way, your workspace root directory would contain `build`, `install`, `log`, and `src` directories in a neat, separate manner.
```suggestion
```

Reply:
At the moment in the directory where the wget is run there are existing other directories running colcon from that directory could lead to errors because it tries to build everything at that level. Maybe add a mkdir fastrtps && cd fastrtps as first command?

Reply:
Yes, you are completely right! I'll add that right away! Thanks for the catch!

Reply:
We are tracking this in #1283